### DISCUSSION TITLE
task dependendies with jreleaser and gradle

### DISCUSSION THREAD

**Discussion by dumischbaenger** [jreleaser/jreleaser#1218] [General]
Hello,

I would like to add a dependency from the jreleaserRelease to shadowJar, but when I add

`jreleaserRelease.dependsOn shadowJar`

to my build file I get an error saying

> Could not get unknown property 'jreleaserRelease' for root project 'jreleasertest' of type org.gradle.api.Project.

Regards,
Bernhard

**Comment by aalmiray** [comment]
This happens because tasks in `v1.4.0` and earlier releases were registered lazily using an `afterEvaluate` project hook. This has been fixed in the upcoming `v1.5.0` with https://github.com/jreleaser/jreleaser/issues/1152 to be released at the end of February, just a few more days to go.
